_layouts/post.html

```diff
@@ -36,8 +36,9 @@ <h2 itemprop="name headline" class="headline">
                     {% if post.image %}
                     <img src="{{ page.image }}" alt="{{ page.title }}" itemprop="image">
                     {% else %}
-                    <meta itemprop="image" content="/assets/files/phalcon-logo-320x180.png">
+                    <meta itemprop="image" content="{{ site.url }}/assets/files/phalcon-logo-320x180.png">
```

Comment:
`{{ site.app.url }}` <- more accurate because we set it.

Reply:
Also check if these files exist in the assets repository (under `phalcon` folder). Then we can cross link them there.

Reply:
There not in the assets available. Already checked. Can add them there if you prefer.